knative-operator/pkg/common/util.go

```diff
@@ -47,13 +48,18 @@ func IngressNamespace(servingNamespace string) string {
 }
 
 // BuildImageOverrideMapFromEnviron creates a map to overrides registry images
-func BuildImageOverrideMapFromEnviron(environ []string) map[string]string {
+func BuildImageOverrideMapFromEnviron(environ []string, scope string) map[string]string {
 	overrideMap := map[string]string{}
 
+	prefix := ImagePrefix
+	if scope != "" {
```

Comment:
I'd prefer we pass `ImagePrefix` in the callers where appropriate rather than silent defaulting. That's more towards my general suggestion though so... maybe not :sweat_smile:

Reply:
I like your suggestion

Reply:
Done